hw/sensor/pkg.yml

```diff
@@ -23,6 +23,12 @@ pkg.author: "Apache Mynewt <dev@mynewt.incubator.apache.org>"
 pkg.homepage: "http://mynewt.apache.org/"
 pkg.keywords:
 
+pkg.deps:
+    - sys/shell
+    - sys/console/full
+    - sys/log/full
+    - sys/stats/full
```

Comment:
Instead of depending on specific implementation of stats, you should use pkg.req_apis.
This allows app builder to either include full stats, or the stub version. The same applies to log, and console.